docker/README.md

```diff
@@ -139,7 +141,7 @@ Expected values :
   * tf-cpu
   * caffe-cpu-tf
   * caffe-tf
-  * caffe2
+  * caffe2 (Only with cpu-xenial.Dockerfile)
```

Comment:
Maybe you mean `gpu-xenial.Dockerfile` ?

Reply:
Oops, that's fixed !